pkgs/top-level/all-packages.nix

```
@@ -3621,6 +3621,10 @@ with pkgs;
 
   clairvoyance = callPackage ../tools/security/clairvoyance { };
 
+  clarissa = callPackage ../tools/networking/clarissa { };
+  clar = callPackage ../tools/networking/clarissa/clar.nix { };
+  clar-oui = callPackage ../tools/networking/clarissa/oui.nix { };
```

Sorry for the late reply, this will work:

```suggestion
  inherit ({
    clarissa = callPackage ../tools/networking/clarissa { };
    clar = callPackage ../tools/networking/clarissa/clar.nix { };
    clar-oui = callPackage ../tools/networking/clarissa/oui.nix { };
  }) clarissa clar clar-oui;
```